games-board/chessx/chessx-1.4.6.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A free and open source chess database application"
+DESCRIPTION="With ChessX you can operate your collection of chess games \
+in many ways: browse, edit, add, organize, analyze, etc."
+HOMEPAGE="https://chessx.sourceforge.net/"
+COPYRIGHT="2005 Michal Rudolf"
+LICENSE="GNU GPL v2"
+REVISION='1'
+SOURCE_URI="https://downloads.sourceforge.net/chessx/chessx/$portVersion/chessx-$portVersion.tgz"
+CHECKSUM_SHA256="ad951910a3bd53f5655f9e85101e7969b536078473e09a2a7dded6ef171a68ed"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	chessx = $portVersion
+	app:chessx = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Multimedia$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Multimedia$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:linguist$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:qmake
```

Comment:
```diff
-	cmd:qmake
+	cmd:qmake$secondaryArchSuffix
```
Explanation: qmake is not available on x86_gcc2 primary arch, and the runtime provided by qt5_x86_devel is in /system/bin/x86/qmake and has a qmake-x86 symlink in /system/bin/ pointing to x86/qmake.
So this means qmake needs a `$secondaryArchSuffix` on x86_gcc2, otherwise the recipe will only build on x86_64.